### DISCUSSION TITLE
Unable to login to my GitHub account via the GitHub desktop app.

### DISCUSSION THREAD

**Discussion by stavrossk** [community/community#21883] [New to GitHub]
<p>Hi, I’m unable to login to my github account through the github desktop app for Windows.</p>
<p>There is an error message that says “Login failed. Make sure you have entered your username and password correctly and try again.”.</p>
<p>I know that my crecendtials are correct because I can login to the github website just fine.</p>
<p>The sync feature also works fine and I can upload my local changes to github website.</p>
<p>I have tried updating the desktop app to the latest versuion and restarting my router and my PC but still the same issue.</p>
<p>My OS is Windows Server 2008 32-bit. Thanks for any help.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/stavrossk">@stavrossk</a>,</p>
<p>When you say that the sync feature is working, do you mean the sync feature in GitHub Desktop? You are able to use this feature to push up your changes without logging into GitHub Desktop?</p>
<p>Thanks!</p>

**Comment by stavrossk** [comment]
<p>Hi <a class="mention" href="/that-pat">@that-pat</a></p>
<p>Yes, I am able to use the Sync feature of the GitHub desktop app to push up my local changes without logging in to my GitHub account via the desktop app.</p>

**Comment by ThirdEyeRose** [comment]
<p><a class="mention" href="/stavrossk">@stavrossk</a>, at which point during the process are you being asked to sign in?</p>

**Comment by stavrossk** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> I’m never asked to sign in. It’s just that when I go to the screen to clone an online repository, it tells me to go to the Settings of the app, add my GitHub account and login. But login never works when I do that. Thanks for your attention.</p>

**Comment by ThirdEyeRose** [comment]
<p><a class="mention" href="/stavrossk">@stavrossk</a>, are you using the <a href="https://desktop.github.com/" rel="nofollow noopener">latest version of GitHub Desktop</a> or a legacy version? And this error only happens when you try to clone a new public repository? I just want to make sure we’re on the same page if I can replicate this situation.</p>

**Comment by stavrossk** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> Yes, I have the latest version of Github Desktop app.</p>
<p>The error always happens when I try to login to my account via the Settings page.</p>
<p>It doesn’t matter what I had been doing before visiting the Settings page (trying to clone an online repository locally or not), it always errors on the login form of the Settings page, even if I try to login immediately after opening the desktop app. Hope this answers your question.</p>

**Comment by stavrossk** [comment, marked as answer]
<p>For some unknown reason after the automatic updating of Github Desktop version, the old version was still present and launched when I clicked the Github Desktop icon in the Start Menu.</p>
<p>My issue was solved after manually downloading and installing the latest version from the Github website. Thanks for the help.</p>
